superset/connectors/sqla/models.py

```diff
@@ -254,9 +254,9 @@ class SqlaTable(Model, BaseDatasource):
     column_class = TableColumn
 
     __tablename__ = 'tables'
-    __table_args__ = (UniqueConstraint('database_id', 'table_name'),)
+    __table_args__ = (UniqueConstraint('database_id', 'schema', 'table_name'),)
 
-    table_name = Column(String(250))
+    table_name = Column(String(127))
```

Comment:
I think this will cause problems. When you create new "Tables" in SQL Lab it will autogenerate very long table names.

Reply:
@fabianmenges unless the database or tab names are significantly long I suspect that [this](https://github.com/apache/incubator-superset/blob/master/superset/assets/src/SqlLab/components/VisualizeModal.jsx#L73) shouldn't generate very long table names.